board/src/test/java/com/example/board/domain/post/controller/PostControllerTest.java

```diff
@@ -0,0 +1,231 @@
+  void createPost() throws Exception {
+
+    //given
+
+    //when
+
+    //then
+    mockMvc.perform(post("/api/posts")
+            .content(objectMapper.writeValueAsString(postCreateRequest))
+            .contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isCreated())
+        .andDo(print())
+        .andDo(document("post-create",
+                requestFields(
+                    fieldWithPath("userId").type(JsonFieldType.NUMBER).description("userId"),
+                    fieldWithPath("title").type(JsonFieldType.STRING).description("title"),
+                    fieldWithPath("content").type(JsonFieldType.STRING).description("content")
+                )
+            )
+        );
+  }
+
+  @Test
+  @DisplayName("성공 - 게시글 단건 조회")
+  void getPost() throws Exception {
+
+    // given
+    Long savedPostId = postResponse.id();
+
+    // when
+
+    // then
+    mockMvc.perform(get("/api/posts/{postId}", savedPostId)
+            .contentType(MediaType.APPLICATION_JSON))
+        .andExpect(status().isOk())
+        .andDo(print())
+        .andDo(document("post-get",
+            responseFields(
+                fieldWithPath("id").type(JsonFieldType.NUMBER).description("id"),
+                fieldWithPath("title").type(JsonFieldType.STRING).description("title"),
+                fieldWithPath("content").type(JsonFieldType.STRING).description("content"),
+                fieldWithPath("createdAt").type(JsonFieldType.STRING).description("createdAt"),
+                fieldWithPath("createdBy").type(JsonFieldType.STRING).description("createdBy"),
+                fieldWithPath("userResponse").type(JsonFieldType.OBJECT)
+                    .description("userResponse"),
+                fieldWithPath("userResponse.id").type(JsonFieldType.NUMBER)
+                    .description("userId"),
+                fieldWithPath("userResponse.name").type(JsonFieldType.STRING)
+                    .description("userName"),
+                fieldWithPath("userResponse.age").type(JsonFieldType.NUMBER).description("userAge"),
+                fieldWithPath("userResponse.hobby").type(JsonFieldType.STRING)
+                    .description("userHobby"),
+                fieldWithPath("userResponse.createdAt").type(JsonFieldType.STRING)
+                    .description("userCreatedAt"),
+                fieldWithPath("userResponse.createdBy").type(JsonFieldType.NULL)
+                    .description("userCreatedBy")
+            )
+        ));
+  }
+
+  @Test
+  @DisplayName("성공 - 게시글 페이징 조회")
+  void getPosts() throws Exception {
+    //given
+    Pageable pageable = PageRequest.of(0, 10);
+
+    //when
+
+    //then
+
+    mockMvc.perform(get("/api/posts")
+            .param("page", String.valueOf(pageable.getPageNumber()))
+            .param("size", String.valueOf(pageable.getPageSize())))
+        .andExpect(status().isOk())
+        .andDo(print())
+        .andDo(document("post-get-page",
+            responseFields(
```

Comment:
쿼리 파라미터에 대한 문서화도 해주면 더 좋을 것 같아요!

Reply:
```suggestion
        .andDo(document("post-get",
        queryParameters(
            parameterWithName("postId").description("게시글 아이디"))
        ),
        ResponseFields(
```
이런식으로 Restdocs에 쿼리 파라미터를 문서화할 수 있습니다~

Reply:
문서화했습니다!
fdf4df705150ade5fc51d45ca11daed5c5e81ff1